src/wix/WixToolset.Converters/WixConverter.cs

```diff
@@ -525,7 +524,7 @@ private void EnsurePrecedingWhitespaceCorrect(XText whitespace, XNode node, int
         {
             if (!WixConverter.LeadingWhitespaceValid(this.IndentationAmount, level, whitespace.Value))
             {
-                var message = testType == ConverterTestType.WhitespacePrecedingEndElementWrong ? "The whitespace preceding this end element is incorrect." : "The whitespace preceding this node is incorrect.";
+                var message = testType == ConverterTestType.WhitespacePrecedingEndElementWrong ? "The whitespace preceding this end element is incorrect." : "The whitespace preceding this comment is incorrect.";
```

Comment:
Is the whitespace always before a comment?

Reply:
No, not in general, but in the cases I was working on, it was.